README.md

```diff
@@ -26,10 +26,34 @@ If you're planning on optimizing your app with ProGuard, make sure that you excl
 
     -keep class com.stripe.** { *; }
 
-You also need to add some configuration options for Gson, which is used by the Stripe bindings to serialize and deserialize JSON data. You can find the recommended ProGuard configuration for Gson [here](https://github.com/google/gson/blob/master/examples/android-proguard-example/proguard.cfg).
-
 ## Usage
 
+### Using the CardInputWidget
+
+You can add a widget to your apps that easily handles the UI states for collecting card data.
+
+First, add the CardInputWidget to your layout.
+
+```xml
+    <com.stripe.android.view.CardInputWidget
+        android:id="@+id/card_input_widget"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        />
```

Comment:
It's not a big deal, but I notice that some of your code blocks in this file are fully dedented (see the example under "createToken"), but these ones are both indented. I'd probably recommend going fully one way or the other.